mlkem/src/native/api.h

```diff
@@ -414,7 +414,7 @@ __contract__(
   requires(memory_no_alias(buf, buflen))
   assigns(memory_slice(r, sizeof(int16_t) * len))
   ensures(return_value == MLK_NATIVE_FUNC_FALLBACK || (0 <= return_value && return_value <= len))
```

Comment:
While at it, we might want to use `==>` here, too, and perhaps consolidate the two `ensures` clauses.